HardwareExperienced.md

```diff
@@ -0,0 +1,87 @@
+| --- | --- | --- |
+| Basic Den hosting | All microcontroller programs plus... | Automated Dynamax Adventures!
+| Day Skipping | Automated purple beam finder | All Microcontroller programs |
+| Egg Hatchers | Den Hosting with stats!  | All Computer Controlled programs |
+| Unattended Shiny Hunts | Autonomous Shiny Hunts! |      |
+
+*The levels of automation build off each other, to do Auto Max Lair you have to proceed trhough Microcontroller and Computer Controlled tutorials. You have been warned.*
+
+# (2) Microcontroller Hardware
+
+**No matter the level of automation you need, start here!**
+
+_Buy the necessary items from A & B._
+
+### A. Buy hardware under the column of what type of Switch you have.
+
+| Switch | Switch Lite |
+| --- | --- |
+| Mini-USB male to USB-A male cable or adapter. ([example 0](https://www.amazon.com/Cmple-Pack-Male-5-Pin-Adapter/dp/B00A1PH0ZW), [example 1](https://www.amazon.com/gp/product/B00P0GI68M)) | Mini-USB male to USB-A male cable or adapter. ([example](https://www.amazon.com/gp/product/B07QJTX59H/)) |
+|     | A USB hub or portable dock. ([example](https://www.amazon.com/gp/product/B07JK9DFKH)) |
+
+### B. Buy *one* of the following Microcontrollers.
+
+Select which column best exemplifies your experience and purchase that microcontroller.
+
+| [Ardunio Leonardo](https://www.amazon.com/gp/product/B0786LJQ8K) | [Teensy2.0](https://www.pjrc.com/store/teensy.html) / [Teensy2.0++](https://www.pjrc.com/store/teensypp.html) | [ProMicro](https://www.amazon.com/gp/product/B08BJNV1J3) |
+| --- | --- | --- | 
+| For beginners | For beginners | Experienced users |
+| I am accustomed to Arduino | I want to easily add programs| I want to automate multiple switches, cheaply |
+| I'm not experienced with electronics | I'm not experienced with electronics | I know my way around electronics |
+
+### C. Verify you have the following
+
+1. A Mini-USB male to USB-A male cable or adaptor (from step (2)A)
+2. A microcontroller (from step (2)2B)
+3. A USB hub or portable dock (if you have a Switch Lite!)
+
+## **If you are only doing Microcontroller Automation proceed to the next step [here](https://github.com/Ensamma/Microcontroller/blob/master/Wiki/Hardware/README.md)**
+If you are doing Computer Controlled or Auto Max Lair automation, proceed below to buy more hardware.
+
+---
+
+# (3) Additional Computer Controlled Hardware
+
+_Buy the necessary items from A, B, C, & D._
+
+### A. Serial UART board
+
+Purchase *one* of the following serial boards
+
+| Adafruit UART  | CP210x Board | 
+| --- | --- |
+| [CP210x controller](https://www.adafruit.com/product/954) | [1 for $8](https://www.amazon.com/dp/B072K3Z3TL) |
+|   | [2 for $8](https://www.amazon.com/gp/product/B07D6LLX19/) |
+|   | [4 for $8](https://www.amazon.com/gp/product/B07T1XR9FT) |
+
+### B. Serial Board Connections
+
+Purchase *one* of the following Board Connections
```

Comment:
Will need to distinguish the Arduinos from the others since the Arduinos only need jumper cables.